Formula/git-imerge.rb

```diff
@@ -16,13 +18,57 @@ class GitImerge < Formula
     sha256 "76aee24eeb5e7615e4cfbd7aaf3aacac8d8729dfcee79d8542a84cbd9b663459" => :el_capitan
   end
 
+  depends_on "python@3.9"
+
+  patch :DATA
+
   def install
-    bin.mkpath
-    # Work around Makefile insisting to write to $(DESTDIR)/etc/bash_completion.d
-    system "make", "install", "DESTDIR=#{prefix}", "PREFIX="
+    virtualenv_install_with_resources
   end
 
   test do
-    system bin/"git-imerge", "-h"
+    system "git", "init"
+    system "git", "config", "user.name", "BrewTestBot"
+    system "git", "config", "user.email", "BrewTestBot@test.com"
+
+    (testpath/"test").write "foo"
+    system "git", "add", "test"
+    system "git", "commit", "-m", "Initial commit"
+
+    system "git", "checkout", "-b", "test"
+    (testpath/"test").append_lines "bar"
+    system "git", "commit", "-m", "Second commit", "test"
+    assert_equal "Already up-to-date.", shell_output("#{bin}/git-imerge merge master").strip
+
+    system "git", "checkout", "master"
+    (testpath/"bar").write "bar"
+    system "git", "add", "bar"
+    system "git", "commit", "-m", "commit bar"
+    system "git", "checkout", "test"
+
+    expected_output = <<~EOS
+      Attempting automerge of 1-1...success.
+      Autofilling 1-1...success.
+      Recording autofilled block MergeState('master', tip1='test', tip2='master', goal='merge')[0:2,0:2].
+      Merge is complete!
+    EOS
+    assert_match expected_output, shell_output("#{bin}/git-imerge merge master 2>&1")
   end
 end
+
```

Comment:
Is this submitted upstream?

Reply:
yes, indeed. PR ref, https://github.com/mhagger/git-imerge/pull/176

Reply:
And it got merged as well, I will add as comments.

Reply:
Can we use the upstream patch then

Reply:
I cannot directly apply the patch (I tried that)

Reply:
@SMillerDev I think we can shoot for the merge for this PR?